pkg/kubernetes/nulecule.js

```diff
@@ -323,6 +396,68 @@ define([
             return;
         };
 
+        function save_provider_config() {
+            var deferred = $.Deferred();
+            deferred.notify(_("saving provider config file ..."));    
+            cockpit.spawn(["/usr/bin/kubectl", "config", "view", "--output=json", "--raw"])
```

Comment:
Instead of launching kubectl again, I would suggest modifying the KubernetesClient so it exposes the config it connected with.